### DISCUSSION TITLE
About Showing Abbreviations on particular semester cards...

### DISCUSSION THREAD

**Discussion by SarveshDevrukhkar** [kunalkeshan/SRMIST-B.Tech-ECE-Notes-2022-24#169] [Ideas]
Hello @kunalkeshan,

Have you observed that the use of abbreviations is from `semester 4` to `semester 7`?
So, unnecessarily it is showing in semesters 1, 2, and 3.

So, what I was saying is what if we add conditions like,
```jsx
if (semester key/count/semester Number === 4, 5, 6, 7) {
show abbreviation otherwise don't.
}
```
Is it necessary or should we keep it as it is?
> :bulb: Let me know what you think about this.

**Comment by kunalkeshan** [comment]
Hey @SAD0XER, yes it does make sense, I feel it's okay to let it be there, and won't affect the way the semesters without any electives are perceived mainly because I feel that it's understandable that all colleges have electives only after the first year.
